spec/features/embedded_touchpoints_spec.rb

```diff
@@ -23,9 +23,13 @@
         context 'default success text' do
           before do
             click_on('Help improve this site') # opens modal
-            expect(page).to have_content('Help improve this site')
 
+            expect(page).to have_content('Help improve this site')
             expect(page).to have_content('Do you have a few minutes to help us test this site?')
+
+            expect(page).to have_content('Page 1')
```

Comment:
Test that page 1 is visible and page 2 and 3 are not.

Reply:
also consider `expect(page).to_not have_content('Page 2')`

Reply:
Changed.